source/actors/player/Item.hx

```diff
@@ -0,0 +1,75 @@
+package actors.player;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.FlxObject;
+import flixel.util.FlxTimer;
+
+class Item extends FlxSprite 
+{
+    private var owner:FlxSprite;
+    private var timer:FlxTimer;
+
+    private var xMod:Float = 0;
+    private var yMod:Float = 0;
+	private var time:FlxTimer;
+    
+    public function new(owner:FlxSprite) {
+        super();
+        makeGraphic(0,0);
```

Comment:
A graphic with dimensions 0,0 causes an exception in the code (the game doesn't crash, but it shows up if you run it in debug mode with `lime test neko -debug`
```suggestion
        makeGraphic(1,1);
```